### DISCUSSION TITLE
Use data that is not defined in columns within table?

### DISCUSSION THREAD

**Discussion by kalaschnik** [TanStack/table#5333] [Q&A]
Hey y'all,

I have data properties that I want to show in the table, yet not in a column. Instead, I want to show a property on hover a label (using title argument).

For example, I've got the following data object

```js
[
    {
      id: "728ed52f",
      amount: 100,
      currency: "USD",
      status: "pending",
      email: "m@example.com",
    },
    {
      id: "489e1d42",
      amount: 125,
      currency: "EUR",
      status: "processing",
      email: "example@gmail.com",
    },
  ];
```

Is there a way to show the currency property as `title` value for the corresponding value within the Amount column? For example:
![PixelSnap 2024-02-06 at 13 28 55](https://github.com/TanStack/table/assets/9831678/7d124338-5cfd-4963-abbf-b5d43a363f36)

Here is a code sandbox:
https://codesandbox.io/p/devbox/vigilant-lumiere-9kvwvw?file=%2Fapp%2Fpage.tsx%3A6%2C1-21%2C5

**Comment by jonahallibone** [comment]
You could make amount an object, like:
```
[
    {
      id: "728ed52f",
      amount: {
        value: 100,
        currency: "USD"
      }
      status: "pending",
      email: "m@example.com",
    },
];
```

Make the accessor a function that looks for amount.value and the in the row display, set the title value of the div to be `row.original.amount.currency`


Alternatively and with the original shape, in column definitions just don't define a column helper for the amount field and access it the same way as above on the row render.
